unikraft/arch/architecture.go

```diff
@@ -15,6 +15,69 @@ import (
 	"kraftkit.sh/unikraft/component"
 )
 
+type ArchitectureName string
+
+const (
+	ArchitectureUnknown = ArchitectureName("unknown")
+	ArchitectureX86_64  = ArchitectureName("x86_64")
+	ArchitectureAmd64   = ArchitectureX86_64
+	ArchitectureArm64   = ArchitectureName("arm64")
+	ArchitectureAarch64 = ArchitectureArm64
+	ArchitectureArm     = ArchitectureName("arm")
+	ArchitectureAarch32 = ArchitectureArm
```

Comment:
Let's keep it simply instead of having aliases:
```go
	ArchitectureUnknown = ArchitectureName("unknown")
	ArchitectureX86_64  = ArchitectureName("x86_64")
	ArchitectureArm64   = ArchitectureName("arm64")
	ArchitectureArm     = ArchitectureName("arm")
```
This is what's in the core